Userland/Libraries/LibSoftGPU/Device.cpp

```diff
@@ -464,6 +465,27 @@ Device::Device(const Gfx::IntSize& size)
     , m_depth_buffer { adopt_own(*new DepthBuffer(size)) }
 {
     m_options.scissor_box = m_render_target->rect();
+
+
+    // Initialize all lights
+    for (auto light_id = 0u; light_id < NUM_LIGHTS; light_id++)
+    {
+        Light light;
+
+        // Light0 has special initial values
+        if (light_id == 0u) {
```

Comment:
Maybe we should let `LibGL` set the initial value for light 0 on initialization of the soft GPU so we can keep `LibSoftGPU` as GL-agnostic as possible?